tests/src/test/scala/system/basic/WskBasicTests.scala

```diff
@@ -732,56 +757,67 @@ class WskBasicTests extends TestHelpers with WskTestHelpers {
   }
 
   it should "reject enable of rule that does not exist" in {
     val name = "nonexistentRule"
-    val stderr = wsk.rule.enable(name, expectedExitCode = NOT_FOUND).stderr
-    stderr should include regex (s"""Unable to enable rule '$name': The requested resource does not exist. \\(code \\d+\\)""")
+    val stderr = wsk.rule.enable(name, expectedExitCode = NotFound.intValue).stderr
+    stderr should include regex (s"""The requested resource does not exist.""")
   }
 
   it should "reject disable of rule that does not exist" in {
     val name = "nonexistentRule"
-    val stderr = wsk.rule.disable(name, expectedExitCode = NOT_FOUND).stderr
-    stderr should include regex (s"""Unable to disable rule '$name': The requested resource does not exist. \\(code \\d+\\)""")
+    val stderr = wsk.rule.disable(name, expectedExitCode = NotFound.intValue).stderr
+    stderr should include regex (s"""The requested resource does not exist.""")
   }
 
   it should "reject status of rule that does not exist" in {
     val name = "nonexistentRule"
-    val stderr = wsk.rule.state(name, expectedExitCode = NOT_FOUND).stderr
-    stderr should include regex (s"""Unable to get status of rule '$name': The requested resource does not exist. \\(code \\d+\\)""")
+    val stderr = wsk.rule.state(name, expectedExitCode = NotFound.intValue).stderr
+    stderr should include regex (s"""The requested resource does not exist.""")
   }
 
   it should "reject get of rule that does not exist" in {
     val name = "nonexistentRule"
-    val stderr = wsk.rule.get(name, expectedExitCode = NOT_FOUND).stderr
-    stderr should include regex (s"""Unable to get rule '$name': The requested resource does not exist. \\(code \\d+\\)""")
+    val stderr = wsk.rule.get(name, expectedExitCode = NotFound.intValue).stderr
+    stderr should include regex (s"""The requested resource does not exist.""")
   }
 
-  behavior of "Wsk Namespace CLI"
+  behavior of "Wsk Namespace REST"
 
   it should "return a list of exactly one namespace" in {
-    val lines = wsk.namespace.list().stdout.lines.toSeq
-    lines should have size 2
-    lines.head shouldBe "namespaces"
-    lines(1).trim should not be empty
+    val lines = wsk.namespace.list()
+    lines.getBodyListString().size shouldBe 1
   }
 
   it should "list entities in default namespace" in {
     // use a fresh wsk props instance that is guaranteed to use
     // the default namespace
-    wsk.namespace.get(expectedExitCode = SUCCESS_EXIT)(WskProps()).stdout should include("default")
+    val result = wsk.namespace.get()(WskProps()).stdout
+    result should include("actions")
+    result should include("packages")
+    result should include("rules")
+    result should include("triggers")
+  }
+
+  it should "not list entities with an invalid namespace" in {
+    val namespace = "fakeNamespace"
+    val stderr = wsk.namespace.get(Some(s"/${namespace}"), expectedExitCode = Forbidden.intValue).stderr
```

Comment:
use action list as this will no longer work soon.

Reply:
I will remove this test cases, since get is not supported by REST implementation.